config/local.js

```diff
@@ -1,3 +1,3 @@
 module.exports = {
-	BE_URL: 'http://localhost:8080'
+	BE_URL: 'http://localhost:8081'
 };
```

Comment:
could you explain this change?

Reply:
Going forward, we will have GS manager as important part for local env setup.
So, all the communication from gs/taro will go to GS manager and not to the BE.
GS manager will run on http://localhost:8081
https://github.com/moddio/gs_manager
The setup steps are exactly the same as BE.